.github/workflows/merge.yml

```diff
@@ -0,0 +1,92 @@
+# Runs whenever a PR is merged:
+# - attempt to backports fixes
+# - upload nightly docs
+#
+# Future:
+# - upload nightly CLI builds
+# - upload nightly vscode extension
+# - upload benchmarks
+# - compute coverge
+#
+# Note that direct commits to master circumvent this workflow!
+
+name: Backport merged pull request
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions:
+  contents: write # so it can comment
+  pull-requests: write # so it can create pull requests
+
+jobs:
+  # Attempt to backport a merged pull request to the latest stable release
```

Comment:
@jkelleyrtp It's great to see you're trying out backport-action 🙌 

By default, backport-action only backports pull requests with a label matching the regex `^backport ([^ ]+)$`. If you want to always backport to a specific branch instead of having to label pull requests, you can use the [`target_branches`](https://github.com/korthout/backport-action?tab=readme-ov-file#target_branches) input.

For example, if you want to always backport to the latest stable release branch, and this branch is called `latest-stable`, you can use:

```yml
uses: korthout/backport-action@v2
with:
  target_branches: 'latest-stable'
```